setup.md

```diff
@@ -21,3 +21,12 @@
 * Each label is stuck on a post-it: yellow = doc, red = bug, green = feature.
 
 * Make some large labels to group related issues (e.g. ggplot2, r-lib) on the wall.
+
+## Icebreaker
+
+The file `bingo.pdf` with source `bingo.qmd` contains instructions for an icebreaker game to help get attendees chatting. To prepare:
+
+* Print 80 copies of the `bingo.pdf` file.
```

Comment:
This assumes that helpers will tag along. I'll track down a printer in the next few days🏄